src/lib/tests/e2e/dashboard.e2e.test.js

```diff
@@ -1,106 +0,0 @@
-/*
```

Comment:
Why was this deleted?

Reply:
yup, this a jest e2e test was present. I saw this deleted in the older PR. So I had also removed it.

Reply:
Ok, why did you choose cypress over the testing we already head? I found the existing tests more readable and better structured, as they could be grouped with `describe` for example and its using the familiar jest syntax of `expect`.

Reply:
@mtrezza I wasn't aware of this jest e2e integration earlier, prior I had used cypress for e2e testing and hence chose cypress. I think jest is used more for unit testing.

Reply:
Well, what is the difference between UI test and unit test?

Reply:
We test individual components in case of unit testing, while e2e tests are used for testing a particular flow.

Reply:
I see what you mean, I've edited my question but you were quicker to answer :)